src/test/java/HomepageTest.java

```diff
@@ -1,45 +1,146 @@
+        softAssertions.assertThat(homepage.getErinBannerBottomText()).isEqualTo("Shop Erin Recommends");
+        softAssertions.assertThat(homepage.getErinBannerLink())
+                .isEqualTo("https://magento.softwaretestingboard.com/collections/erin-recommends.html");
+        softAssertions.assertThat(homepage.getErinBannerImageSource())
+                .isEqualTo("https://magento.softwaretestingboard.com/pub/media/wysiwyg/home/home-erin.jpg");
+        softAssertions.assertAll();
+    }
+
+    @Test
+    public void homepage_EcoFriendlyBanner_HasTextAndLinkCorrectlyDisplayed() {
+        //given
+        Homepage homepage = new Homepage(driver);
+
+        //then
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(homepage.isEcoFriendlyBannerDisplayed()).isTrue();
+        softAssertions.assertThat(homepage.isEcoFriendlyLinkEnabled()).isTrue();
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerTopText()).isEqualTo("Twice around, twice as nice");
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerMainText()).isEqualTo("Find conscientious, comfy clothing in our eco-friendly collection");
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerBottomText()).isEqualTo("Shop Eco-Friendly");
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerLink())
+                .isEqualTo("https://magento.softwaretestingboard.com/collections/eco-friendly.html");
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerImageSource())
+                .isEqualTo("https://magento.softwaretestingboard.com/pub/media/wysiwyg/home/home-eco.jpg");
+        softAssertions.assertAll();
+    }
+
+    @Test
+    public void homepage_PerformanceBanner_HasTextAndLinkCorrectlyDisplayed() {
+        //given
+        Homepage homepage = new Homepage(driver);
+
+        //then
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(homepage.isPerformanceBannerDisplayed()).isTrue();
+        softAssertions.assertThat(homepage.isPerformanceLinkEnabled()).isTrue();
+        softAssertions.assertThat(homepage.getPerformanceBannerTopText()).isEqualTo("Science meets performance");
+        softAssertions.assertThat(homepage.getPerformanceBannerMainText()).isEqualTo("Wicking to raingear, Luma covers you");
+        softAssertions.assertThat(homepage.getPerformanceBannerBottomText()).isEqualTo("Shop Performance");
+        softAssertions.assertThat(homepage.getPerformanceBannerLink())
+                .isEqualTo("https://magento.softwaretestingboard.com/collections/performance-fabrics.html");
+        softAssertions.assertThat(homepage.getPerformanceBannerImageSource())
+                .isEqualTo("https://magento.softwaretestingboard.com/pub/media/wysiwyg/home/home-performance.jpg");
+        softAssertions.assertAll();
     }
```

Comment:
Why Hot Sellers section test is not added. Please add it.
It does not need to contain checks for all products attribute.
So you can add test in which following elements are validated:
- Hot Sellers text
- Here is what`s trending on Luma right now text
- presence of 6 top sellers product
- for each of product: presence of image, product-item-details, and product name